docs/20_smart-contracts/10_getting-started/10_dune-guide/index.md

```diff
@@ -0,0 +1,634 @@
+---------------------------------------------------------------------------------
+first       |    Y    |    Y     | 127.0.0.1:8888 | 0.0.0.0:9876 | 127.0.0.1:8080
+```
+
+You can also check that the node is listening to RPC API requests by running the following command:
+
+```shell
+$ dune -- cleos get info
+```
+
+The command returns information about the local EOS blockchain just launched:
+
+```json
+{
+  "server_version": "3f9cb99e",
+  "chain_id": "8a34ec7df1b8cd06ff4a8abbaa7cc50300823350cadc59ab296cb00d104d2b8f",
+  "head_block_num": 42987,
+  "last_irreversible_block_num": 42986,
+  "last_irreversible_block_id": "0000a7eac208ba842a890f28f8afa26a692760f1d824f618168f0bafa54ed28f",
+  "head_block_id": "0000a7ebb0dd4e78c5283ec67eb01a803673e23f77c57dee2733e1a170418dd9",
+  "head_block_time": "2023-02-17T06:14:57.000",
+  "head_block_producer": "eosio",
+  "virtual_block_cpu_limit": 200000000,
+  "virtual_block_net_limit": 1048576000,
+  "block_cpu_limit": 199900,
+  "block_net_limit": 1048576,
+  "server_version_string": "v3.2.1",
+  "fork_db_head_block_num": 42987,
+  "fork_db_head_block_id": "0000a7ebb0dd4e78c5283ec67eb01a803673e23f77c57dee2733e1a170418dd9",
+  "server_full_version_string": "v3.2.1-3f9cb99e068da8581036d212d74fb56144ef0fc4",
+  "total_cpu_weight": 0,
+  "total_net_weight": 0,
+  "earliest_available_block_num": 1,
+  "last_irreversible_block_time": "2023-02-17T06:14:56.500"
+}
+```
+
+You can also check the `get_info` endpoint directly exposed by the `chain_api_plugin` in your browser:
+
+```shell
+http://localhost:8888/v1/chain/get_info
+```
+
+Or you can use `curl` or `wget` to send an HTTP GET request directly:
+
+```shell
+curl http://localhost:8888/v1/chain/get_info
+```
+
+In both cases, the result is similar to the JSON output above.
+
+> ℹ️ Node Bootstrapping  
+Bootstrapping a node becomes necessary when either the system contracts must be deployed, or some functionality must be activated by a protocol feature. For simplicitly, node bootstrapping will be showcased in another guide when required.
+
+### Create Test Accounts
+
+EOS accounts give ownership to stakeholders in an EOS blockchain. To deploy a smart contract, you need to create an EOS account to host it. Accounts also represent the actors that send actions to a smart contract. For more information about EOS accounts, visit the [Accounts](../20_accounts/index.md) guide.
+
+> ℹ️ DUNE Account Creation  
+DUNE simplifies account creation by abstracting key management. When an account gets created, DUNE also creates public/private key pairs for its two default permissions, `owner` and `active`. Therefore, if you use DUNE then key creation is not required during [Wallet Setup](#wallet-setup).
+
+For this guide, you will create three accounts: `hello`, then `bob` and `alice`. The first account, `hello`, will be used in this guide to deploy your `Hello World` smart contract. The last two, `bob` and `alice`, are test accounts to be used in later guides.
```

Comment:
"The last two, `bob` and `alice`, are test accounts to be used in later guides."

Guides should not be asking readers to do work for other guides. If another guide needs `bob` or `alice` to be created, they should be created in that guide instead of asking more work of a developer in this guide.

Reply:
Yes I debated on this as well. Removed `bob` and `alice` account creation.